document/core/syntax/instructions.rst

```diff
@@ -285,28 +258,30 @@ SIMD instructions provide basic operations over :ref:`values <syntax-value>` of
      \K{i64x2.}\K{mul} \\&&|&
      \K{i8x16.}\AVGR\K{\_u} ~|~
      \K{i16x8.}\AVGR\K{\_u} \\&&|&
-     \K{i16x8.}\EXTMUL\K{\_low}\K{\_i8x16\_}\sx ~|~
-     \K{i16x8.}\EXTMUL\K{\_high}\K{\_i8x16\_}\sx \\&&|&
-     \K{i32x4.}\EXTMUL\K{\_low}\K{\_i16x8\_}\sx ~|~
-     \K{i32x4.}\EXTMUL\K{\_high}\K{\_i16x8\_}\sx \\&&|&
-     \K{i64x2.}\EXTMUL\K{\_low}\K{\_i32x4\_}\sx ~|~
-     \K{i64x2.}\EXTMUL\K{\_high}\K{\_i32x4\_}\sx \\&&|&
+     \K{i16x8.}\EXTMUL\K{\_}\half\K{\_i8x16\_}\sx ~|~
+     \K{i32x4.}\EXTMUL\K{\_}\half\K{\_i16x8\_}\sx ~|~
+     \K{i64x2.}\EXTMUL\K{\_}\half\K{\_i32x4\_}\sx ~|~
```

Comment:
Can we throw a new line between the \EXTMULs and \EXTADDPAIRWISE, this line currently goes beyond the gray box.
<img width="1335" alt="Screen Shot 2021-08-12 at 9 18 54 AM" src="https://user-images.githubusercontent.com/1749303/129231937-c5051ffe-c1e1-4088-b443-d0a7288ffc92.png">

Reply:
Done.